tests/test_simulation.py

```diff
@@ -611,7 +632,67 @@ def test_dephasing():
         )
 
 
-def test_add_config():
+def test_depolarizing():
+    np.random.seed(123)
+    reg = Register.from_coordinates([(0, 0)], prefix="q")
+    seq = Sequence(reg, Chadoq2)
+    seq.declare_channel("ch0", "rydberg_global")
+    duration = 2500
+    pulse = Pulse.ConstantPulse(duration, np.pi, 0, 0)
+    seq.add(pulse, "ch0")
+    sim = Simulation(
+        seq, sampling_rate=0.01, config=SimConfig(noise="depolarizing")
+    )
+    assert sim.run().sample_final_state() == Counter({"0": 587, "1": 413})
+    assert len(sim._collapse_ops) != 0
+    with pytest.warns(UserWarning, match="first-order"):
+        reg = Register.from_coordinates([(0, 0), (0, 6)], prefix="q")
+        seq2 = Sequence(reg, Chadoq2)
+        seq2.declare_channel("ch0", "rydberg_global")
+        seq2.add(pulse, "ch0")
+        sim = Simulation(
+            seq2,
+            sampling_rate=0.01,
+            config=SimConfig(noise="depolarizing", depolarizing_prob=0.5),
+        )
+
+
+def test_eff_noise(matrices):
+    np.random.seed(123)
+    reg = Register.from_coordinates([(0, 0)], prefix="q")
+    seq = Sequence(reg, Chadoq2)
+    seq.declare_channel("ch0", "rydberg_global")
+    duration = 2500
+    pulse = Pulse.ConstantPulse(duration, np.pi, 0, 0)
+    seq.add(pulse, "ch0")
+    sim = Simulation(
+        seq,
+        sampling_rate=0.01,
+        config=SimConfig(
+            noise="eff_noise",
+            eff_noise_opers=[matrices["I"], matrices["Z"]],
+            eff_noise_probs=[0.95, 0.05],
+        ),
+    )
```

Comment:
If I'm not mistaken, this should be the same as dephasing with `p=0.10`. Why don't you run a simulation with that config and check that the final states match?